ci/pipeline

```diff
@@ -246,36 +279,78 @@ def jenkins(): Unit = {
  * @return Version and artifact description of Marathon build.
  */
 @main
-def release(requestVersion: String, gitSha: String, latest: Boolean = false): Unit = {
-  val releaseVersion = withoutVersionPrefix(requestVersion)
-  val tagVersion = s"v$releaseVersion"
+def release(args: String*): Unit = {
+  case class Config(
+    targets: List[ReleaseTarget] = Nil,
+    runTests: Boolean = true,
+    checkout: Option[String] = None)
+
+  val parser = new scopt.OptionParser[Config]("scopt") {
+    head("pipeline release")
+    opt[Boolean]("run-tests").required().action { (runTests, c) =>
+      c.copy(runTests = runTests)
+    }
+    opt[String]("checkout").action {
+      case ("", c) => c.copy(checkout = None)
+      case (checkout, c) => c.copy(checkout = Some(checkout))
+    }
+    arg[String]("targets...").unbounded.action { (target, c) =>
+      ReleaseTarget.all.find(_.name == target) match {
+        case Some(rt) => c.copy(targets = rt :: c.targets)
+        case None =>
+          throw new RuntimeException(s"${target} is not a known release target; valid options are:\n* ${ReleaseTarget.all.map(_.name).mkString("\n* ")}")
+      }
+    }
+  }
 
-  require(ValidVersionRegex.findFirstMatchIn(tagVersion).nonEmpty,
-    s"tagName should match regex ${ValidVersionRegex}")
+  val config = parser.parse(args, Config()).getOrElse {
+    sys.exit(1)
+    ???
+  }
+
+  config.checkout.foreach { gitSha =>
+    %('git, "checkout", gitSha)
+  }
 
-  (tagVersion, requestVersion)
+  // if (pushTag) {
```

Comment:
Why is this commented out?

Reply:
Did not change this; Jenkins does not have access to push the tag. Some day we would like to have jenkins tag and push on success.

Reply:
I think this could easily be done with the Github API.